mkFit/MkFitter.cc

```diff
@@ -501,19 +501,23 @@ void MkFitter::FindCandidates(std::vector<Hit>& lay_hits, int firstHit, int last
 
 
 void MkFitter::GetHitRange(std::vector<std::vector<BinInfo> >& segmentMapLay_, int beg, int end,
-                           const float etaDet, int& firstHit, int& lastHit)
+                           int& firstHit, int& lastHit)
 {
 
     int itrack = 0;
 
 
+    // what exactly is this function doing?  why are you calling the first simtrack in the event to get the eta??
```

Comment:
not sure if this kind of comments should go into the devel branch.
we should rather discuss in person what these methods do